jetty-core/jetty-deploy/src/main/config/modules/deployer-standard.mod

```diff
@@ -1,5 +1,6 @@
 [description]
-This module enables the direct Deployer, which simply adds and starts a context.
+This module enables the standard Deployer, which adds and starts ContextHandler
+instances produced by the deployment-scanner module.
```

Comment:
```suggestion
instances. This module is used by the deployment-scanner module to deploy the web applications it discovers.
```

Reply:
I like it less. The fact that ContextHandler instances are produced by deployment-scanner is lost.